From 096e33743158e0e8c04d60d01cc66e2945d79777 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 27 Nov 2018 16:52:35 +0800
Subject: [PATCH] From 557ca399f4b3a397f20bb147ec6dc4ab9732dd1e Mon Sep 17
 00:00:00 2001 From: Khem Raj <raj.khem@gmail.com> Date: Fri, 31 Mar 2017
 19:12:10 -0700 Subject: [PATCH] Replace __S_IEXEC with S_IEXEC

S_IEXEC is portable

Signed-off-by: Khem Raj <raj.khem@gmail.com>

update patch to version 5.1.5
Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
Upstream-Status: Pending

 daemon/lookup.c        | 6 +++---
 modules/lookup_multi.c | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/daemon/lookup.c
+++ b/daemon/lookup.c
@@ -397,7 +397,7 @@ static int read_file_source_instance(str
 		return NSS_STATUS_NOTFOUND;
 	}
 
-	if (st.st_mode & __S_IEXEC)
+	if (st.st_mode & S_IEXEC)
 		type = src_prog;
 	else
 		type = src_file;
@@ -930,7 +930,7 @@ static int lookup_name_file_source_insta
 		return NSS_STATUS_NOTFOUND;
 	}
 
-	if (st.st_mode & __S_IEXEC)
+	if (st.st_mode & S_IEXEC)
 		type = src_prog;
 	else
 		type = src_file;
@@ -1077,7 +1077,7 @@ static struct map_source *lookup_get_map
 	if (!S_ISREG(st.st_mode))
 		return NULL;
 
-	if (st.st_mode & __S_IEXEC)
+	if (st.st_mode & S_IEXEC)
 		type = "program";
 	else
 		type = "file";
--- a/modules/lookup_multi.c
+++ b/modules/lookup_multi.c
@@ -247,7 +247,7 @@ static struct lookup_mod *nss_open_looku
 				continue;
 			}
 
-			if (st.st_mode & __S_IEXEC)
+			if (st.st_mode & S_IEXEC)
 				type = src_prog;
 			else
 				type = src_file;
